src/parse_structure.ml

```diff
@@ -63,11 +68,15 @@ let simplify_lambda lambda =
 #endif
 
 let transl_exp expr =
+#if OCAML_VERSION >= (4, 12, 0)
+  Translcore.transl_exp ~scopes:Debuginfo.Scoped_location.empty_scopes expr
+#else
 #if OCAML_VERSION >= (4, 11, 0)
```

Comment:
I think you can use the `#elif` statement here to avoid the two `#endif` at the end.

Reply:
Indeed :-)